app/src/main/res/layout/activity_main.xml

```diff
@@ -8,70 +8,70 @@
     tools:context=".MainActivity">
 
 
-
     <TextView
-        android:id="@+id/textView2"
+        android:id="@+id/txt_menu"
         android:layout_width="match_parent"
         android:layout_height="50dp"
         android:background="@color/themeColor"
-        android:textSize="20dp"
-        android:textStyle="bold"
-        android:textColor="@color/white"
         android:gravity="center"
-        android:text="Menu" />
+        android:text="@string/txt_menu_text"
```

Comment:
Add string from `strings.xml` instead of hardcoded text

Reply:
Definitely a good idea.  Good for internationalization.